CODEOWNERS

```diff
@@ -294,6 +294,8 @@ extensions/filters/http/oauth2 @derekargueta @snowp
 /*/extensions/load_balancing_policies/maglev @wbpcode @UNOWNED
 # Early header mutation
 /*/extensions/http/early_header_mutation/header_mutation @wbpcode @UNOWNED
+# Header mutation
+/*/extensions/filters/http/header_mutation @wbpcode @UNOWNED
```

Comment:
Should this be starting life without an additional maintainer to provide coverage?

Reply:
The code/logic of this filter is very simple. I am not sure if it's necessary to have an additional maintainer to help? 🤔

Reply:
Sure, but bus factor of one isn't great for ownership and https://github.com/envoyproxy/envoy/blob/main/EXTENSION_POLICY.md suggests two.

Reply:
Also we could have this other maintainer review this initial implementation to spin them up.

Reply:
Get it. I wonder do you have free bandwidth for this? 😸  If you have free time, then we needn't to bother third maintainer. Thanks.

Reply:
Yeah, I actually have a need to deep dive in some related parts of Envoy, so I can do this if there isn't a more interested party :)